app/src/org/commcare/utils/FormUploadUtil.java

```diff
@@ -316,9 +309,9 @@ private static boolean buildMultipartEntity(MultipartEntity entity,
         return true;
     }
 
-    private static int addPartToEntity(MultipartEntity entity, File f, ContentBody fb) {
-        if (fb.getContentLength() <= MAX_BYTES) {
-            entity.addPart(f.getName(), fb);
+    private static int addPartToEntity(List<MultipartBody.Part> parts, File f, String contentType) {
+        if (f.length() <= MAX_BYTES) {
```

Comment:
Informational (IE: If you see this and it makes you think of something, good, but I think it's fine it not):

Slightly pedantic, but this was (I believe) intentionally targeting the content length previously, think? I think the equivalent in this case would likely be 
```
part = createFilePart(name,file,type)
if(part.contentLength < MAX_BYTES)
  parts.add(part)
...
```
but I don't have a specific scenario in my head where that's likely to affect things unless the body itself ends up being bigger/smaller by a significant amount, which could occur in very small part due to metadata (but not enough for us to care).

Reply:
So I checked the implementation for `FileBody:: getContentLength()` and it just returns `file.length()`. So I think we should be good here.

Reply:
Great! Checked off.